runtime/src/main/java/org/corfudb/runtime/exceptions/TrimmedException.java

```diff
@@ -1,10 +1,22 @@
 package org.corfudb.runtime.exceptions;
 
+import lombok.Getter;
+import lombok.Setter;
+import org.corfudb.runtime.view.Address;
+
 /**
  * This exception is thrown when a client tries to read an address
  * that has been trimmed.
  */
 public class TrimmedException extends LogUnitException {
+    @Getter
+    @Setter
+    long address = Address.NON_ADDRESS;
+
+    @Getter
+    @Setter
+    long snapshot = Address.NON_ADDRESS;
```

Comment:
No, I don't agree. Whether a trimmed exception is recoverable or not has nothing to do with the number of retries.